src/main/content/_assets/js/collections.js

```diff
@@ -20,20 +21,37 @@ function getCollectionData(instanceName){
     if(typeof instanceName === "undefined"){
         return;
     }
-
     return fetch(`/api/auth/kabanero/${instanceName}/collections/list`)
         .then(function(response) {
             return response.json();
         })
+        .then(function(data){
+            return updateCollectionView(data);
+        })
+        .then(() =>{
+            return instanceName;
+        })
```

Comment:
these lines would not be needed once the first comment in the promise chain is addressed